client/src/components/AddBookModal/AddBookModal.tsx

```diff
@@ -107,30 +122,38 @@ const AddBookModal: React.FC<AddBookModalProps> = ({closeModalFunc, addBookFunct
   } else if(modalContent === 5) {
     content = 
       <div>
-        <p>Is this book a must read?</p>
-          <div>
+        <h2>Is this book a must read?</h2>
+          <div className='addbook-modal-grand-wrapper__pop-out-container__checkbox-div'>
             <input type="checkbox" value='yes' name='starRead' onChange={(e) => handleChange(e)} checked={userInput.starRead ? true: false}/>
             <label htmlFor="yes">star read</label>
           </div>
       </div>
   }
 
+  console.log(userInput)
+
   return (
-    <div data-testid='add-book-modal'>
+    <div data-testid='add-book-modal' className='addbook-modal-grand-wrapper'>
+      <div 
+        className='addbook-modal-grand-wrapper__greyed-background-div'
+        onClick={() => closeModalFunc()}
+      ></div>
       <AiFillCloseCircle  data-testid="close-modal-icon" onClick={() => closeModalFunc()}/>
-      {content}
-      <div>
-        {modalContent !== 1 && <button onClick={() => {setModalContent(modalContent - 1)}}>back</button>}
-        {modalContent !== 5 ?  <button onClick={() => {
-          // if(modalContent === 2 && !userInput.review)  return
-          setModalContent(modalContent + 1)
-        }
-        }
-        >next</button> : <button onClick={() => {
-          addBookFunction(userInput)
-          closeModalFunc()
-        }
-        }>add book</button>}
+      <div className='addbook-modal-grand-wrapper__pop-out-container'>
+        {content}
+        <div className='addbook-modal-grand-wrapper__pop-out-container__button-div'>
+          {modalContent !== 1 && <button onClick={() => {setModalContent(modalContent - 1)}}>back</button>}
+          {modalContent !== 5 ?  <button onClick={() => {
```

Comment:
This is...not so fun to reason about. See comment above about changing the modal content state away from a numerical sequence and using strings instead. 
That way you could have a handleBack and handleForward functions which use if / elses to update the step depending on what the current one is. Or, it could look a bit like:

```jsx
const backStepLookup = {
  whatDidYouThink: 'whenDidYouRead'
  // other steps...
 }

const handleBack = (step) => setModalStep(backStepLookup[step])
  
```

Reply:
Might be also be worth looking up what state machines are - not necessarily to use one here, but to understand why you *might want to use one* for this sort of thing